Update Node Iterator Docs (#16639)
mdn/content: files/en-us/web/api/document/createnodeiterator/index.md

---
{{APIRef("DOM")}}

> The `Document.createNodeIterator()` method returns a newly created [`NodeIterator`](/en-US/docs/Web/API/NodeIterator) object.

Reviewer: ```suggestion
The **`Document.createNodeIterator()**` method returns a new [`NodeIterator`](/en-US/docs/Web/API/NodeIterator) object.
```

Reviewer 2: **`Document.createNodeIterator()`** 
bold and code got mixed

Reviewer: Ooops, fixed.

> The `Document.createNodeIterator()` creator method returns a newly created [`NodeIterator`](/en-US/docs/Web/API/NodeIterator) object.

Reviewer 2: ```suggestion
The `Document.createNodeIterator()` method returns a newly created [`NodeIterator`](/en-US/docs/Web/API/NodeIterator) object.
```

- `filter` {{optional_inline}}
  - : A callback function or an object with an `acceptNode()` method. The function or method will be called for each node in the subtree based at root which is accepted as included by the whatToShow flag to determine whether or not to include it in the list of iterable nodes. The method should return one of `NodeFilter.FILTER_ACCEPT`, `NodeFilter.FILTER_REJECT`, or `NodeFilter.FILTER_SKIP`. See the [Example](#example).

> ### Return value

Reviewer 2: ```suggestion
### Return value

```
